.github/workflows/ci.yaml

```diff
@@ -0,0 +1,22 @@
+name: CI
+
+on: [push, pull_request]
```

Comment:
This triggers on pushes to feature branches too. You need to restrict it to pushes from the default branch.

Reply:
Is that technically a problem that the bot would run on feature branches too?

Reply:
Done that though.